src/AuthProvider.tsx

```diff
@@ -23,25 +24,31 @@ export const AuthProvider: React.FC<PropsWithChildren> = ({children}) => {
       login: async (email: string) => {
         try {
           console.log(`Trying to login with email: ${email}`);
+          sampleRUM('chat:login', { source: 'attempt' });
           setState({token: state.token, loading: true});
           const token = await magic.auth.loginWithMagicLink({email});
           setState({token, loading: false});
           console.log(`Logged in with email: ${email}`, JSON.stringify(token));
+          sampleRUM('chat:login', { source: 'success' });
         } catch (e) {
           console.error(`Failed to login with email: ${email}`, e);
           setState({token: state.token, loading: false});
+          sampleRUM('chat:login', { source: 'failed' });
```

Comment:
```suggestion
          sampleRUM('chat:login-failed', { source: 'failed' });
```

This is a different, distinct checkpoint